File: ios/Runner/AdConfig.swift
struct AdConfig {
    static let appId             = "4582335cc3af56be" // 智探眼_iOS
    // posId 由 Flutter 端 ad_config.dart 统一管理，通过 MethodChannel 参数传入
    // 以下仅作回退兜底，正常不会用到
    static let splashPosId       = "99b82c48067f2084" // 智探眼_iOS-开屏
    static let rewardedPosId     = "bd569504ee7f0fcc" // 智探眼_iOS-激励视频
    static let bannerPosId       = "37a403bd9c243d90" // 智探眼_iOS-横幅
    static let interstitialPosId = "f39df5130e18f3d0" // 智探眼_iOS-插屏
    static let detailBannerPosId = "d6c5e2397e02e837" // 智探眼_iOS-原生混合
}

struct AdChannels {
    static let method = "com.billtrack/ad_method"
    static let event  = "com.billtrack/ad_event"
}

File: android/app/src/main/kotlin/com/jileduo/finance/AdConfig.kt
package com.jileduo.finance

internal object AdConfig {
    // posId 以 Flutter 端 ad_config.dart 为准(经 MethodChannel 传入),下方仅回退兜底
    const val APP_ID               = "e3a729330c8ca9a4" // 智探眼_Android
    const val CHANNEL              = "default"
    const val SPLASH_POS_ID        = "198b01ffa638de2d" // 智探眼_Android-开屏
    const val REWARDED_POS_ID      = "d48de64ce7fa3da8" // 智探眼_Android-激励视频
    const val BANNER_POS_ID        = "90b9d6c08cfdf27d" // 智探眼_Android-横幅
    const val INTERSTITIAL_POS_ID  = "69f84f4e4aba54b1" // 智探眼_Android-插屏
    const val DETAIL_BANNER_POS_ID = "c6d90d9e869f7098" // 智探眼_Android-原生混合
}
